Makefile

```diff
@@ -51,8 +51,8 @@ downloads/drive:
 
 ########################################################################################################
 
-CHAT_VERSION = 2.0.1
-CHAT_LINK = "https://s3-eu-west-1.amazonaws.com/zextras-artifacts/openchat/20_Jul_2018_OP-CPB-34/openchat.tgz"
+CHAT_VERSION = 1.0.12
+CHAT_LINK = "https://files.zimbra.com/downloads//openchat.tgz"
```

Comment:
As mentioned before, not a fan of having files on files.zimbra.com where you can't work out what version they are.  What's wrong with just renaming the file `openchat-1.0.12.tgz` ?